### ISSUE TITLE
Load only one file

### ISSUE THREAD

**Issue by raphink** [raphink/narcissus#1]
Using `aug_transform`, load only one file.

In order to achieve this, store a `augeasLens` field in the type, with a `default` tag:

``` go
type Passwd struct {
  augeasPath string `default:"/files/etc/passwd"`
  augeasLens string `default:"Passwd.lns"`
}
```

**Comment by raphink** [comment]
This is [done in creds-unsealer](https://github.com/camptocamp/creds-unsealer/blob/master/providers/ovh.go#L65-L78) with:

```go
	err = aug.Transform("IniFile.lns_loose", o.OutputPath, false)
	if err != nil {
		return fmt.Errorf("failed to set up Augeas transform: %s", err)
	}

	err = aug.Load()
	if err != nil {
		return fmt.Errorf("failed to load Augeas tree: %s", err)
	}

	augErr, _ := aug.Get("/augeas/files" + o.OutputPath + "/message")
	if augErr != "" {
		return fmt.Errorf("failed to load file with Augeas: %s", augErr)
	}
```

**Comment by raphink** [comment]
Implemented as of 950142198b3d847adc2221cf0fc1f50df76b9258
